### DISCUSSION TITLE
Um dos grandes problemas que devs encontram no MVP de clientes.

### DISCUSSION THREAD

**Discussion by romulosardinha** [frontendbr/forum#1708] [Discussão]
Texto sobre relato de problema ao empreendedor criar um MVP e o que os devs podem fazer para ajudar a melhorar o resultado.

https://medium.com/@ross.txt/o-problema-do-modelo-mvp-para-startups-e7c5ac1fa467

**Comment by felipe-heredia** [comment]
### O grande problema das Startups.

O grande problemas dessas novas empresas com ideias mirabolantes (que na verdade é algo que já existe mas na cabeça do "empreendedor" é algo totalmente inovador) é querer começar de cima, com tudo, 100% já feito.

_A construção de uma casa é por partes, não se começa uma casa pintando as paredes._

Querem começar um produto, já pensando nele totalmente pronto, porém não é assim que funciona.

Se quer criar uma nova empresa, talvez mirabolante, precisa começar do zero, no exemplo utilizado a Uber, primeiro deveria pensar: Mapa, Pagamentos e Aproximação. O resto, com o tempo seria evolunido, quase ninguém utiliza o chat do Uber, depois de algumas semanas, deveria ser **pensando** em melhorar animações e adicionar novas funcionalidades.

**Comment by romulosardinha** [comment]
@beam2232 o que vc achou sobre a proposta de utilizar tempo/grana como uma variável a fim de determinar o que é esta primeira construção?

**Comment by kavalcante** [comment]
Estou empreendendo há alguns anos já. Além do que foi abordado no post ser verdade. Outro problema que eu vejo muito recorrente, é o total desconhecimento das pessoas de negócio sobre tecnologia (sobre como construir um produto com tecnologia).

A vantagem do mundo da tecnologia, é que ela é barata em relação a outras formas de empreender. Pra montar uma industria, você gastaria no mínimo algumas vezes mais do que você gasta para construir uma startup. Isso faz com que seja mais acessível, o que é ótimo, mas também não estimula muito as pessoas a conhecer o que é ter uma empresa de tecnologia.

Mas eu entendo esses movimentos como naturais, a alta mortalidade das startups está de alguma maneira relacionada a isso também. Empreendedores que dão mais certo, são os que faliram algumas vezes. Pois esses e vários outros erros são aprendidos (ou deveriam) na primeira experiência.

**Comment by romulosardinha** [comment]
@kavalcante Concordo. A facilidade de criar aumenta a taxa de mortalidade. Mas assim como uma seleção natural, possibilita ideas que são bem 'fora da curva' terem a chance de sucesso e talvez mudar nosso modo de pensar.
